tests/README.md

```diff
@@ -3,14 +3,16 @@
 This project is using Open MCT's e2e-as-a-dependency model. To learn more, please see the official documentation on the [Official README](https://github.com/nasa/openmct/blob/master/e2e/README.md)
 
 ## How to Run Locally
+To run the tests, we recommend the following workflow which bridges two separate github repos:
+yamcs/quickstart and openmct-yamcs (this one).
 
 1. `git clone yamcs/quickstart`
-2. `cd docker`
-3. `make all`
-4. Sanity test that yamcs is up with `npm run wait-for-yamcs`
-5. `cd openmct-yamcs`
-6. `npm install`
-7. `npx playwright@1.29.0 install`
+2. `cd docker` in yamcs/quickstart
+3. `make all` in yamcs/quickstart
+4. `cd openmct-yamcs` to move out of yamcs/quickstart
+5. `npm install` in openmct-yamcs
+6. `npx playwright@1.29.0 install chromium` in openmct-yamcs
```

Comment:
This still should be updated to `1.32.3`? Or no?

Reply:
oh good catch